### PR TITLE
update for fable 3. Slider is not working

### PR DESCRIPTION
**Pull request by whitetigle**
Seems to work on my side. 
Version is using Parcel instead of webpack.
Removed download of netcore in build script.
Build has not been tested yet. 
Slider handler.value stuck to 0.0.

### CODE REVIEW

File: showcase/src/App.fs [hunk 1]
```diff
@@ -87,14 +93,15 @@ module Main =
...
-        stats.``begin``()
+        //stats.``begin``()
         #endif
 
         ui.Prepare()
 
         if ui.Window(window1) then
             ui.Label(sprintf "Clicked: %i times" buttonCounter, Center)
             if ui.Button("Click me") then
+                printfn "Click"
```
Comment (MangelMaxime) [inline showcase/src/App.fs]: Can you please remove this debug `printfn`

**Comment by MangelMaxime** [comment]
For me the slider is working but not the combo component but for the combo it can be because it is inside another window and I probably forget to take the window deplacement in account

image

**Comment by whitetigle** [comment]
Yup slide is working because I rewrote the Clamp function. 
Ok for the combo.

**Comment by MangelMaxime** [comment]
Ah yes, I didn't saw this change. 

Combo has been fixed in master branch

**Pull request merged**